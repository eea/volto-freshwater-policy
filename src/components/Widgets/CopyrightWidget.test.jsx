import React from 'react';
import { render } from '@testing-library/react';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';

import CopyrightWidget from './CopyrightWidget';
const mockStore = configureStore();

describe('renders a copyright widget component', () => {
  it('basic', () => {
    const store = mockStore({
      intl: {
        locale: 'en',
        messages: {},
      },
    });

    const { asFragment } = render(
      <Provider store={store}>
        <CopyrightWidget
          id="copyright"
          title="CopyRight"
          fieldSet="default"
          onChange={() => {}}
        />
      </Provider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
