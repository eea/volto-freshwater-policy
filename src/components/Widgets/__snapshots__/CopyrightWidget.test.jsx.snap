// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a copyright widget component basic 1`] = `
<DocumentFragment>
  <div
    class="inline field copyright-widget-wrapper field-wrapper-copyright"
  >
    <div
      class="ui grid"
    >
      <div
        class="stretched row"
      >
        <div
          class="four wide column"
        >
          <div
            class="wrapper"
          >
            <label
              for="field-copyright"
              id="fieldset-default-field-label-copyright"
            >
              CopyRight
            </label>
          </div>
        </div>
        <div
          class="eight wide column"
        >
          <div
            class="ui checked radio checkbox"
          >
            <input
              checked=""
              class="hidden"
              readonly=""
              tabindex="0"
              type="radio"
              value="EEA"
            />
            <label>
              EEA Copyright Creative Commons CC-by licence
            </label>
          </div>
          <div
            class="ui radio checkbox"
          >
            <input
              class="hidden"
              readonly=""
              tabindex="0"
              type="radio"
              value="other"
            />
            <label>
              Other
            </label>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
